docs/source/install/index.rst

```diff
@@ -9,6 +9,12 @@ single system, or you can follow the manual instructions for your OS.
 
 Here's an overview of the options:
 
+* **Vagrant / Virtual Appliance:** Vagrant / OVA is a quick and easy way to try StackStorm.
+  It's already pre-installed, tested and shipped as a virtual image and so saves your time going
+  through time-consuming installation and configuration steps. Works best as a testing,
+  pack development or demo system and recommended to get familiar with StackStorm platform.
+  ``vagrant init stackstorm/st2 && vagrant up`` is all you need to get started.
```

Comment:
I don't like having this as the top-level option. We can have "Virtual Appliance" as first option, but I don't like having Vagrant like that, especially when most people don't have Vagrant installed, and are going to use whatever IaaS they are familiar with (local vSphere, AWS, etc)

Maybe we need to separate this into "Quick demo" and "Production" options.

Reply:
If we mean OpenSource & DevOps oriented-people, - Vagrant is definitely is used and known.

I really would like to push pre-packaged Vagrant as a first thing here, so people meeting with StackStorm for the first time have easiest, fastest and most reproducible option without going left with other installation methods they don't need yet or not ready to try yet.

I believe this approach will save our time, support efforts and improve first overall StackStorm user experience.